clash-lib/src/Clash/Netlist/Util.hs

```diff
@@ -513,6 +513,7 @@ representableType builtInTranslation reprs stringRepresentable m =
 typeSize :: HWType
          -> Int
 typeSize (Void {}) = 0
+typeSize FileType = 32
```

Comment:
Suspicious, but it checks out. Maybe add a reference to page 287 of IEEE 1364-2005, just so we know why it's not equal to "word width" of the platform you're running on.

Reply:
Added a comment